src/disco/bank/fd_txncache.c

```diff
@@ -345,25 +346,66 @@ fd_txncache_delete( void * shtc ) {
   return (void *)tc;
 }
 
+static void
+fd_txncache_remove_blockcache_idx( fd_txncache_t * tc,
+                                   ulong idx ) {
+  tc->blockcache[ idx ].lowest_slot = ULONG_MAX;
+  memcpy( tc->txnpages_free+tc->txnpages_free_cnt, tc->blockcache[ idx ].pages, tc->blockcache[ idx ].pages_cnt*sizeof(ushort) );
+  tc->txnpages_free_cnt += tc->blockcache[ idx ].pages_cnt;
+
+  /* Check if this a probed entry or not. If this is an entry inserted
+     at the intended "hash index" we still need to check if there are
+     any probed entries and shift one to this index. */
+  ulong hash_idx = FD_LOAD(ulong, tc->blockcache[ idx ].blockhash);
+  if( hash_idx%tc->live_slots_max != idx ) return;
+
+  /* We linear search for an entry which hashed to the same index and move it to
```

Comment:
This fix isn't correct. Relocating an probed entry might then cause a gap that needs to itself have a reprobed item moved into it.